<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergerControlWidget</class>
 <widget class="QWidget" name="MergerControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="15" column="1">
    <widget class="EuclidianMoveParametersControlWidget" name="pos1ControlWidget" native="true">
     <property name="toolTip">
      <string>pos1</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="EuclidianMoveParametersControlWidget" name="pos2ControlWidget" native="true">
     <property name="toolTip">
      <string>pos2</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="EuclidianMoveParametersControlWidget" name="pos4ControlWidget" native="true">
     <property name="toolTip">
      <string>pos4</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="EuclidianMoveParametersControlWidget" name="pos3ControlWidget" native="true">
     <property name="toolTip">
      <string>pos3</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="frameToUndistSpinBox">
     <property name="toolTip">
      <string>frameToUndist</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Out Size H</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>Undist Focal</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="bilinearCheckBox">
     <property name="toolTip">
      <string>bilinear</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_0">
     <property name="text">
      <string>undistMethod</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="undistMethodComboBox">
     <property name="toolTip">
      <string>undistMethod</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square Table</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radial Table</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD Table</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Loaded Camera</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="distortionOvershootSpinBox">
     <property name="toolTip">
      <string>Distortion Overshoot</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>showMask</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="sensorWidthSpinBox">
     <property name="toolTip">
      <string>Sensor Width</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="showMaskCheckBox">
     <property name="toolTip">
      <string>showMask</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>bilinear</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Separate View</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="undistFocalSpinBox">
     <property name="toolTip">
      <string>Undist Focal</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>400.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="separateViewCheckBox">
     <property name="toolTip">
      <string>Separate View</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sensor Width</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Distortion Overshoot</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Draw Car</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="drawCarCheckBox">
     <property name="toolTip">
      <string>Draw Car</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>frameToUndist</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QDoubleSpinBox" name="fOV1SpinBox">
     <property name="toolTip">
      <string>FOV1</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>deg</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>FOV3</string>
     </property>
    </widget>
   </item>
   <item row="26" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>FOV1</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>pos2</string>
     </property>
    </widget>
   </item>
   <item row="26" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>switch3</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDoubleSpinBox" name="fOV2SpinBox">
     <property name="toolTip">
      <string>FOV2</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>deg</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>switch2</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>ground Z</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="groundZSpinBox">
     <property name="toolTip">
      <string>ground Z</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <double>-99999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>-20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Out Phy Size L</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="outPhySizeWSpinBox">
     <property name="toolTip">
      <string>Out Phy Size W</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>dm</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QCheckBox" name="switch3CheckBox">
     <property name="toolTip">
      <string>switch3</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QDoubleSpinBox" name="fOV4SpinBox">
     <property name="toolTip">
      <string>FOV4</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>deg</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QCheckBox" name="switch2CheckBox">
     <property name="toolTip">
      <string>switch2</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>switch4</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>switch1</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>pos1</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="outSizeHSpinBox">
     <property name="toolTip">
      <string>Out Size H</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="minimum">
      <double>10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="switch1CheckBox">
     <property name="toolTip">
      <string>switch1</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>pos3</string>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>pos4</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="outPhySizeLSpinBox">
     <property name="toolTip">
      <string>Out Phy Size L</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>dm</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>FOV2</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>FOV4</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Out Phy Size W</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QDoubleSpinBox" name="fOV3SpinBox">
     <property name="toolTip">
      <string>FOV3</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string>deg</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QCheckBox" name="switch4CheckBox">
     <property name="toolTip">
      <string>switch4</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QPushButton" name="saveRemapButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Save remap</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <customwidgets>
  <customwidget>
   <class>EuclidianMoveParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>euclidianMoveParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>undistMethodComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undistFocalSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensorWidthSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distortionOvershootSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMaskCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bilinearCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>separateViewCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawCarCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frameToUndistSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outSizeHSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outPhySizeLSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outPhySizeWSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groundZSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOV1SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switch1CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos1ControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOV2SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switch2CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos2ControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOV3SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switch3CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos3ControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOV4SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switch4CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos4ControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>MergerControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
</ui>
